### PR TITLE
don't crash when bitcoind isn't yet up

### PR DESCRIPTION
**Pull request by ProofOfKeags**

### CODE REVIEW

File: configurator/src/main.rs [hunk 1]
```diff
@@ -344,6 +344,20 @@ fn main() -> Result<(), anyhow::Error> {
                 )
             }
         };
+
+        loop {
```
Comment (dr-bonez) [inline configurator/src/main.rs:348]: do we want to add a timeout to this?

**Review by dr-bonez — APPROVED** [review]

File: configurator/src/main.rs (hunk 1 above)
Comment (ProofOfKeags) [reply configurator/src/main.rs:348]: I was told to do it *forever*

Comment (chrisguida) [reply configurator/src/main.rs:348]: forever sounds good

**Comment by chrisguida** [comment]
Needs rebase, looks good

**Pull request merged**